src/components/Guilds/Header/WalletButton.tsx

```diff
@@ -0,0 +1,91 @@
+import styled from 'styled-components';
+import { observer } from 'mobx-react';
+import { isDesktop } from 'react-device-detect';
+import WalletModal from 'components/WalletModal';
+import { getChains, injected } from 'provider/connectors';
+import { useEffect, useState } from 'react';
+import { useRpcUrls } from 'provider/providerHooks';
+import { useContext } from '../../../contexts';
+import { Button, IconButton } from '../common/Button';
+import dxIcon from '../../../assets/images/dxdao-icon.svg';
+
+const ButtonIcon = styled.img`
+  height: 1.1rem;
+  width: 1.1rem;
+`;

```

Comment:
This component is repeated across WalletButton and NetworkButton, I suggest that you should create a common ButtonIcon component if it does not already exist

Reply:
you can also create a base CSS and import that (instead of a component).

Reply:
This is addressed in my other PR (#449) which handles ENS avatar loading.